docs/4-Commands/5-kool-run.md

```diff
@@ -1,10 +1,10 @@
 ## kool run
 
-Runs a custom command defined at kool.yaml
+Runs a custom command defined at kool.yaml file in the current working directory or in the kool folder of the user's home directory.
```

Comment:
Another documentation file we should update I guess is the `kool.yml` session on `docs/1-Getting-Started/4-How-it-works.md`